_data/projects/kiel.yml

```
@@ -0,0 +1,12 @@
+name: Kiel
+desc: Kotlin way of building RecyclerView Adapter 🧩.
+site: https://ibrahimyilmaz.github.io/kiel/
+tags:
+  - kotlin
+  - recyclerview
+  - android
+  - androiddev
+  - viewholder
+upforgrabs:
+  name: up-for-grabs
+  link: https://github.com/ibrahimyilmaz/kiel/labels/up%20for%20grabs
```

@ibrahimyilmaz I think you mistyped the Label URL
```suggestion
  link: https://github.com/ibrahimyilmaz/kiel/labels/up-for-grabs
```

---

It is not miss typing. It is written so in the up for grabs official documentation!
Up-for-grabs or up for grabs, the difference is this. So your thinking is false but anyway i did your suggestion.

---

```
upforgrabs:
  name: up for grabs
  link: https://github.com/username/project/labels/up%20for%20grabs
```
You meant this in https://github.com/up-for-grabs/up-for-grabs.net/blob/gh-pages/docs/list-a-project.md

here you did not see that it was `name: up for grabs` where as you used ` name: up-for-grabs`

---

@ibrahimyilmaz

---

So what I should do now?